uk.co.compucorp.civicrm.tasksassignments/js/test/crm-tasks-workflows/decorators/route.decorator.spec.js

```diff
@@ -0,0 +1,81 @@
+/* eslint-env jasmine */
+
+(function () {
+  'use strict';
+
+  describe('RouteDecorator', function () {
+    var $route, crmApiSpy;
+
+    beforeEach(module('ngRoute', 'crm-tasks-workflows.decorators', function ($routeProvider) {
+      // This happens before the configuration phase:
```

Comment:
The comment would be much more valuable if "this happens" is replaced with an actual explanation on what's going on.